### ISSUE TITLE
Move identity buttons for creation and deletion above the identities list

### ISSUE THREAD

**Issue by Cwpute** [roundcube/roundcubemail#9657]
### Prerequisites

- [X] I have [searched](https://github.com/roundcube/roundcubemail/issues?q=is%3Aissue) for duplicate or closed feature requests
- [X] I have [searched](https://plugins.roundcube.net/) for plugins that provide already provide the feature

### Proposal

Instead of having the `Create` and `Delete` buttons for identities appear above the identity details in the rightmost panel, have them appear above the identity list in the center panel.

### Motivation and context

I was looking at the identity panel trying to find where to create a new one, and after a dozen seconds clicking, i finally found them, above the rightmost pannel, somewhere i didn't expect them to show up. I know, there is not much in that window, but it's early monday, please bear with me 🙏
To me it doesn't feel right to have these buttons there, as they don't interact directly with the button below. You'll not be creating a new input field, nor will be deleting any: you'll be creating or deleting a whole new identity.
![image](https://github.com/user-attachments/assets/be90aef1-052a-436f-80ec-f70ed11f115d)

**Comment by alecpl** [comment]
This will create inconsistency with other places. So, I'm against it.

**Comment by Cwpute** [comment]
> This will create inconsistency with other places. So, I'm against it.
It would be true, and i would agree if there wasn't inconsistencies already, supporting my suggestion.
I only found out about them recently, sorry for that.

I discovered the issue i was raising was not caused by my own personal preference only, but most certainly by the current design choice being inconsistent:
![example](https://i.postimg.cc/dtD7qBS2/Screenshot-20241004-223228.png)
As you can see on the main screen where the user is presented with their mailboxes and the according mails, actions listed above the center panel all act on the list presented under it, and actions presented under the rightmost panel all act on the list presented below. This is probably the screen where most users would spend most of their tine, and you'd expect to find that design choice i pointed to reproduced in other places in Mailcube

But this isn't the case. As demonstrated in my original message, on the identities screen, actions above the leftmost panel do **not** act on the list below, but on the list presented in center panel. This is also true for the Filters, Folders and the other sections in the Parameters.
In the Filter section a especialy this is the most strange as you get a drop-down menu to add edit and delete filter-groups that are listed below (which is perfectly fine), but on the right panel the buttons that show on top still do **not** act upon the information listed below, but rather on a second *type* of item in the center panel, that being filters contained in filter-groups.

So basically, you've got:
1. the main screen showing emails that displays butons that act on the information presented below them
2. sections like Identities in which buttons on the right act on the list in the center
3. the Filter section that does *both*, having button in the center panel that act on the information below **and** buttons on the right that **do not** act on the information below, but rather the information in the center

These are inconsistencies in the design that can make a user feel lost.
